src/AppBundle/EventListener/CheckSQLiteEventSubscriber.php

```diff
@@ -57,4 +62,24 @@ public function handleDatabaseExceptions(ConsoleExceptionEvent $event)
             }
         }
     }
+
+    /**
+     * This method is triggered when kernel exception occurs. And checks if sqlite extension is enabled.
+     *
+     * @param GetResponseForExceptionEvent $event
+     */
+    public function onKernelException(GetResponseForExceptionEvent $event)
+    {
+        $exception = $event->getException();
+        // Since any exception thrown during a Twig template rendering is wrapped in a Twig_Error_Runtime.
+        // We must get the original exception.
+        $previousException = $exception->getPrevious();
+
+        // Driver exception may happen in controller or in twig template rendering
+        $isDriverException = ($exception instanceof DriverException || $previousException instanceof DriverException);
```

Comment:
@uikolas, you could use a platform name:
```php
$entityManager->getConnection()->getDatabasePlatform()->getName() === 'sqlite'
```
See https://github.com/doctrine/dbal/blob/master/lib/Doctrine/DBAL/Platforms/SqlitePlatform.php#L511